.github/workflows/check-testnet.yml

```diff
@@ -11,6 +11,7 @@ jobs:
   check-spec-version:
     name: Check spec_version bump
     runs-on: SubtensorCI
+    if: ${{ !contains(github.event.pull_request.labels.*.name, 'no-spec-version-bump') }}
```

Comment:
Do we have something for check mainnet too?

Reply:
that's `check-finney` and it's already there in fact.